packages/svelte/tests/runtime-runes/samples/async-derived-module/_config.js

```diff
@@ -14,17 +14,6 @@ export default test({
 		const [reset, a, b, increment] = target.querySelectorAll('button');
 
 		a.click();
-
-		// TODO why is this necessary? why isn't `await tick()` enough?
-		await Promise.resolve();
-		await Promise.resolve();
-		await Promise.resolve();
-		await Promise.resolve();
-		await Promise.resolve();
-		await Promise.resolve();
-		await Promise.resolve();
-		await Promise.resolve();
-		flushSync();
```

Comment:
:laughing: I saw this while I was working on async SSR and I was really hopeful we could murder it at some point